script/bootstrap

```diff
@@ -15,3 +15,6 @@ if [ -f .env.${NODE_ENV} ]; then
 fi
 
 npm install
+
+# Make sure mocha and coffee are in the PATH
+export PATH="node_modules/.bin:$PATH"
```

Comment:
This will stop working after the user closes their terminal, so probably better to use the relative path (`./node_modules/.bin/mocha`, etc.) from the scripts. Good call though!